src/components/atoms/empty-state.md

```diff
@@ -0,0 +1,23 @@
+The `icon` property is required, and should be set to a valid icon type. (Same as the `type`
+property in the `<Icon>` component.) Any children of the component will be rendered underneath
+the icon.
+
```

Comment:
check out the source of [Input documentation](https://github.com/auth0/cosmos/blob/master/src/components/atoms/input.md) to see how to add the props table, I'll add it to the example [`_box` component](https://github.com/auth0/cosmos/blob/master/src/components/atoms/_box.js) as well and add more to the [contributing guide](https://github.com/auth0/cosmos/blob/master/contributing.md).

Reply:
Ah, thanks! I think I just copied (I think) the `<Tabs>` component, and didn't realize there was a table like that available. I'll update the docs.

Reply:
I noticed a few issues using this:

![screen shot 2018-01-05 at 4 13 41 pm](https://user-images.githubusercontent.com/1576/34628734-6e570260-f233-11e7-9557-e3dbe0a210d3.png)

1. Using quotes around the default value will pull those quotes into the prop value. (Using double quotes actually causes a syntax error to be displayed in the docs.)
2. I'm not sure we really want a default value for `icon`, but I get an error message if I don't supply one.
3. Maybe the `colors.grayMedium` token should be resolved to its hex value? It's definitely confusing since it appears here as a string.

Reply:
1. Ouch, let me see what I can do, value handling in the prop table is pretty fragile right now.

2. You can get around that easily by changing your code in the example.

```jsx
<EmptyState icon="help">...</EmptyState>
``

3. I'm still not sure if this is the right way to go: https://github.com/auth0/cosmos/pull/155#issuecomment-355660219